internal/venice-common/src/main/java/com/linkedin/venice/controllerapi/transport/GrpcRequestResponseConverter.java

```diff
@@ -85,6 +85,22 @@ public static void sendErrorResponse(
     responseObserver.onError(StatusProto.toStatusRuntimeException(status));
   }
 
+  public static void sendErrorResponse(
+      Code code,
+      ControllerGrpcErrorType errorType,
+      Exception exception,
+      String clusterName,
+      String storeName,
+      StreamObserver<?> responseObserver) {
+    sendErrorResponse(
+        code,
+        errorType,
+        exception != null ? exception.getMessage() : "",
```

Comment:
can getMessage be null?

Reply:
yeah it can be. Hence we explicitly check non-nullability of message in overridden method `sendErrorResponse`